src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -1137,7 +1137,8 @@
     </Property>
     <Property Name="IsRef" Type="bool">
       <Comments>
-        <summary>Is result a managed reference</summary>
+        <value><see langword="true"/> iff the result is a reference to a struct or class. Not to be confused with <see cref="Type"/>.<see cref="ITypeSymbol.IsReferenceType"/>.</value>
+        <remarks>This occurs in C# for ternaries whose branches use <see langword="ref"/>.</remarks>
```

Comment:
I'm fine with the addition of the remark section, but I think I'd prefer a slight rewording of the original summary, rather than this change. Something like `True if the result is by-reference. I don't think we want to include the works `struct` or `class` here. I also don't think that cref will format correctly.

Reply:
I didn't see a lot of 'iff' in our documentation.

Reply:
```suggestion
        <value><see langword="true"/> if the result is by-reference. Not to be confused with <c>op.Type.IsReferenceType</c>; by-ref can be used with either.</value>
        <remarks>This occurs in C# for ternaries whose branches use <see langword="ref"/>.</remarks>
```
?

(You were right about the links not working, at least in VS Code w/ dev kit plugin.)

Reply:
I don't think we really need the second sentence at all, being perfectly honest.